bakery/src/tasks/upload-single.js

```diff
@@ -0,0 +1,71 @@
+const dedent = require('dedent')
```

Comment:
Since this is a git task, can we pull the shell script out into `bakery/src/scripts` like the rest of the `*-single` tasks?